package.json

```diff
@@ -9,6 +9,7 @@
     "test": "jest"
   },
   "devDependencies": {
+    "@nrwl/nx-darwin-x64": "^15.9.3",
```

Comment:
i'm on M1 chip, should i install nx-darwin-arm64?

Reply:
Yeah ... seems like we need nx-drawin-arm64.
BTW, my env is also M1